spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/DiscoveryBootstrapProperties.java

```diff
@@ -171,7 +176,10 @@ public int getHealthCheckInterval() {
   }
 
   public void setHealthCheckInterval(int healthCheckInterval) {
-    this.healthCheckInterval = healthCheckInterval;
+    if (Math.min(healthCheckInterval, HEALTH_CHECK_MAX_INTERVAL) ==
```

Comment:
Why not `HEALTH_CHECK_MIN_INTERVAL<=healthCheckInterval && HEALTH_CHECK_MAX_INTERVAL>= healthCheckInterval ` ?

Reply:
OK